Versioning.md

```diff
@@ -19,38 +19,38 @@ must remain backwards compatible. Internal types are allowed to break.
 
 ### ABI Stability
 
-Refer [ABI Policy](./docs/abi-policy.md) for more details. To summarise
-* ABI stability is guaranted for API.
-* ABI stability is not guaranteed for SDK. C++ In case of ABI breaking changes, instead of bumping up the majore version, new `inline namespace` version will be crearted, and both old API and new API would be made available simultaneously.
+Refer to the [ABI Policy](./docs/abi-policy.md) for more details. To summarise
+* ABI stability is guaranteed for the API.
+* ABI stability is not guaranteed for the SDK. In case of ABI breaking changes, instead of bumping up the major version, a new `inline namespace` version will be created, and both old API and new API would be made available simultaneously.
```

Comment:
It is ambiguous what these "ABI breaking changes" are referring to. Are these ABI breaking changes to the SDK?  or ABI breaking changes to the API?

Is the plan really to keep two copies of the code (approximately) forever after an ABI breaking change?

Reply:
`ABI breaking changes` refers to SDK here. Will reword it properly. 

> Is the plan really to keep two copies of the code (approximately) forever after an ABI breaking change?

Yes, that's the plan for now, but will be revisiting this before going for GA.